docs/config.md

```diff
@@ -227,6 +227,9 @@ cpu_only_composing_models: <comma-delimited-string-list>
 # Enables the searching of request rate (instead of concurrency)
 [ request_rate_search_enable: <bool> | default: false]
 
+# Always report GPU metrics
```

Comment:
This comment in a .md file doesn't add any value. Can you mention the case where it would be valuable?

Reply:
Done